src/filtering/gaussian.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np 
+
+def gaussian_filter(size_degradation_filter: ndarray, sigma: float) -> ndarray:
+    ''' Gaussian filter
+    :param size_degradation_filter: defines the lateral size of the kernel/filter, default 5
+    :param sigma: standard deviation (dispersion) of the Gaussian distribution
+    :return matrix with a filter [size_degradation_filter x size_degradation_filter] to be used in convolution operations
+    '''
+    arx = np.arange((-size_degradation_filter // 2) + 1.0, (size_degradation_filter // 2) + 1.0)
+    x, y = np.meshgrid(arx, arx)
+    filt = np.exp(-(1/2) * (np.square(x) + np.square(y)) / np.square(sigma))
+
+    return filt / np.sum(filt)
```

Comment:
In `bilateral_filter`, the output is the image after applying the filter. For consistency, I was thinking about `gaussian_filter` offer the same interface, so that the output is the image after applying the filter as well, instead of the filter itself. Tell me what do you think. And thanks for the contribution!

Reply:
Also, perhaps we could apply the Gaussian on frequency domain instead of spatial domain for speeding up things a little bit.